frontend/src/global_styles/layout/_toolbar.sass

```diff
@@ -165,15 +165,15 @@ $nm-color-success-background: #d8fdd1
   input[type=number],
   select
     @include toolbar-element-sizing
-    border: 1px solid var(--toolbar-item--border-color)
+    border: 1px solid var(--button-default-borderColor-rest)
 
   .button,
   .toolbar-input--affix
-    background: var(--toolbar-item--bg-color)
+    background: var(--button-default-bgColor-rest)
 
     &.-pressed,
     &:active
-      background: var(--toolbar-item--bg-color-pressed)
+      background: var(--control-transparent-bgColor-active)
```

Comment:
I'd suggest to use `--button-default-bgColor-active` here instead

Reply:
Hi @HDinger 
I got the variable from GitHub toolbar items styles.

Reply:
I see the `--button-default-bgColor-active` in GitHub as well

<img width="1602" alt="Bildschirmfoto 2024-08-09 um 13 21 46" src="https://github.com/user-attachments/assets/53e0fbe2-17d3-41fe-b66c-ddd2caa3253f">